src/oss-find-squats-lib/MutateExtension.cs

```diff
@@ -105,22 +105,25 @@ public static class MutateExtension
 
             // Check to see if it is a scoped npm package to generate candidates for.
             bool isScoped = purl.Namespace.IsNotBlank() && purl.Type.Equals("npm", StringComparison.OrdinalIgnoreCase);
-            string nameToMutate = isScoped ? (purl.Namespace!).Substring(1) : purl.Name;
+            string nameToMutate = isScoped ? purl.Namespace : purl.Name;
```

Comment:
Why is this change necessary?

Reply:
This change will no longer remove the @ from the front of the namespace.  Won't that break the mutations?

Reply:
PackageUrl stores the namespaces without the @ so for `@angular/core`, `nameToMutate` was being set as `ngular`